src/docs/guide/interactivity.md

```diff
@@ -7,4 +7,4 @@ order: 7
 show_guide: true
 ---
 
-Coming soon.
+Coming soon. See the [cursor component docs](../components/cursor.html) in the meantime.
```

Comment:
eh it's fine for now; I've gotta just tighten up `<code>` to not add so much `padding`, esp when inside `<a>`s